server/src/services/backup.service.ts

```diff
@@ -101,14 +101,23 @@ export class BackupService extends BaseService {
       `immich-db-backup-${Date.now()}.sql.gz.tmp`,
     );
 
+    const databaseVersion = await this.databaseRepository.getPostgresVersion();
+    const databaseMajorVersion = Number.parseInt(databaseVersion.split('.')[0], 10);
```

Comment:
It's semver, right? Should we use the semver package instead then?

Reply:
Moved to checking a range with semver